src/services/JWTService.ts

```diff
@@ -19,6 +19,11 @@ class JWTService {
             throw new AuthorizationError("Azure configuration is not valid.")
         }
 
+        // Check if the role is correct
+        if (!decodedToken.payload.roles && !decodedToken.payload.roles.includes("CVSFullAccess")) {
```

Comment:
might be worth having this as case insensitive match (if not already). Also for now its okay, but be aware later this would mature into list of roles which are allowed access to various bits of CVS microservices. So would be some form of mapping config which allows you to decide which request is allowed access to what